lib/active_admin/views/components/table_for.rb

```diff
@@ -115,10 +115,8 @@ def render_data(data, item)
       end
 
       def is_boolean?(data, item)
-        if item.respond_to? :has_attribute?
-          item.has_attribute?(data) &&
-            item.column_for_attribute(data) &&
-            item.column_for_attribute(data).type == :boolean
+        if item.class.respond_to? :columns_hash
+          column = item.class.columns_hash[data.to_s] and column.type == :boolean
```

Comment:
@seanlinsley does the test covers the difference between `&&` and `and`? Not that some will change it and the test don't show it.